javaslang/src/main/java/javaslang/collection/CharSeq.java

```diff
@@ -803,7 +803,7 @@ public CharSeq reverse() {
 
     @Override
     public IndexedSeq<Character> scan(Character zero, BiFunction<? super Character, ? super Character, ? extends Character> operation) {
-        return scanLeft(zero, operation);
+        return Collections.scanLeft(this, zero, operation, Iterator::toCharSeq);
```

Comment:
Awesome, that was fast :-)
Can the method return CharSeq instead of IndexedSeq&lt;Character&gt;?